### ISSUE TITLE
C-s failed after TEST_END and fails the test unexpectedly

### ISSUE THREAD

**Issue by juliayakovlev** [scylladb/scylla-cluster-tests#4223]
Test: `cloud-longevity-small-data-set-1h-gcp`
Test name: `longevity_test.LongevityTest.test_custom_time`
Test config file(s):
- scylla_cloud_nemesis_small_set.yaml (Siren repo)
- scylla_cloud.yaml (Siren repo)

This c-s failure happens after `TEST_END`:
```
< t:2021-12-14 09:18:45,588 f:file_logger.py  l:85   c:sdcm.sct_events.file_logger p:INFO  > 2021-12-14 09:18:45.584: (InfoEvent Severity.NORMAL) period_type=not-set event_id=652cea1c-f1da-4da2-96f9-8b240f13eae6
: message=TEST_END

< t:2021-12-14 09:18:47,004 f:base.py         l:146  c:RemoteCmdRunner      p:ERROR > Error executing command: "echo TAG: loader_idx:0-cpu_idx:0-keyspace_idx:1; STRESS_TEST_MARKER=A8SJICP9PXNEEJWQW2GL; cassandra
-stress write n=400000 cl=QUORUM -port jmx=6868 -mode native cql3  user=scylla password=cpdk74HLhbR0aGi -schema keyspace=ks_truncate 'replication(factor=6)' -log interval=5 -node 35.237.56.244 -errors skip-unsupported-columns"; Exit status: -1
```

So it should not fail the test

Restore Monitor Stack command: `$ hydra investigate show-monitor 6d94093a-85f8-4bee-8f8f-156c3d113ec7`
Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=6d94093a-85f8-4bee-8f8f-156c3d113ec7)
Show all stored logs command: `$ hydra investigate show-logs 6d94093a-85f8-4bee-8f8f-156c3d113ec7`

Test id: `6d94093a-85f8-4bee-8f8f-156c3d113ec7`

Logs:
db-cluster - [https://cloudius-jenkins-test.s3.amazonaws.com/6d94093a-85f8-4bee-8f8f-156c3d113ec7/20211214_092514/db-cluster-6d94093a.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/6d94093a-85f8-4bee-8f8f-156c3d113ec7/20211214_092514/db-cluster-6d94093a.tar.gz)

sct-runner - [https://cloudius-jenkins-test.s3.amazonaws.com/6d94093a-85f8-4bee-8f8f-156c3d113ec7/20211214_092514/sct-runner-6d94093a.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/6d94093a-85f8-4bee-8f8f-156c3d113ec7/20211214_092514/sct-runner-6d94093a.tar.gz)

[Jenkins job URL](https://jenkins.scylladb.com/job/siren-tests/job/longevity-tests/job/cloud-longevity-small-data-set-1h-gcp/81/)

**Comment by bentsi** [comment]
we don't experience this anymore, it was fixed by Dmitriy in eae84def

**Comment by fgelcer** [comment]
we are actually seeing it once in a while, on `branch-2021.1`, and because i expect it will be EOL soon, i won't re-open this issue
